MotionMark/tests/resources/main.js

```diff
@@ -59,6 +59,11 @@ Sampler = Utilities.createClass(
     }
 });
 
+const sampleTimeIndex = 0;
+const sampleComplexityIndex = 1;
+const sampleFrameLengthIndex = 2;
+const sampleSmoothedFrameLengthIndex = 3;
```

Comment:
I think this change is confusing. The `Controller._sampler` has only three arrays. The last one which has `index = 2` is for the smoothed-frame-length. The `controllerSamples` we create in `_processControllerSamples()` is a SampleData which has a set of four datums. The last one `index = 3` is for the smoothed-frame-length also.

If I want to access the smoothed-frame-length in the smaller data will it be:

```
this._sampler.samples[2][i] 
```

or will it be:

```
this._sampler.samples[sampleSmoothedFrameLengthIndex][i]
```

The data structures are unfortunately error prone and complicated.

Reply:
What if I only add constants for:
```
const sampleTimeIndex = 0;
const sampleComplexityIndex = 1;
```